index.html

```diff
@@ -48,6 +48,13 @@ <h1 class="heading__text">
         </div>
       </div>
     </section>
+    <section class="popup">
+      <div class="popup__container--closed">
+        <h2 class="popup__text--title">Sorry!</h2>
+        <p class="popup__text--content">Our drink monkeys couldn't find anything. Here is our suggestion for tonight!</p>
+        <button class="popup__button">Let's GIT it!!!</button>
```

Comment:
Love a git pun. Not sure we should have it in the final site, but I appreciate that it's here now. :)